src/CodeGen.fs

```diff
@@ -1203,4 +1229,118 @@ type {clientName}(url: string, httpClient: HttpClient) =
 let sampleFSharpGraphqlClient projectName clientName errorType members serializer =
     match serializer with
     | SerializerType.System -> sampleFSharpSystemGraphqlClient projectName clientName errorType members
-    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
\ No newline at end of file
+    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
+
+let sampleFableGraphqlClientFsi projectName clientName =
+        $"""namespace {projectName}
+
+open System.Net.Http
+open Fable.SimpleHttp
+open Fable.SimpleJson
+
+type {clientName} =
+    class
+        /// <summary>Creates {clientName} specifying <see href="T:HttpClient">HttpClient</see> instance</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization <b>you must</b> add
+        /// <see href="T:Fable.SimpleJson.FableJsonConverter">FableJsonConverter</see>
+        /// from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a> NuGet package yourself
+        /// </remarks>
+        /// <param name="url">GraphQL endpoint URL</param>
+        new: url: string * headers: Header list -> SpotifyGraphqlClient
```

Comment:
Hardcoded `SpotifyGraphqlClient` name, this depends on the project name. Please fix